aerialist/px4/obstacle.py

```diff
@@ -113,3 +113,13 @@ def from_coordinates_multiple(cls, coordinates: List[float]):
         for i in range(0, len(coordinates), 7):
             obst.append(Obstacle.from_coordinates(coordinates[i : i + 7]))
         return obst
+
+    @classmethod
+    def from_dict_list(cls, obstacle_list):
```

Comment:
maybe better to have this method for only one obstacle, then move the loop to the calling method. It would be easier to re-use that way. 
Alsom try to put the type indicators in the parameteres where possible.
```suggestion
    def from_dict(cls, obstacle:dict):
```

Reply:
Done